ganga/GangaCore/__init__.py

```diff
@@ -891,7 +891,7 @@ def filefilter(fn):
     pwd_nam = pwd.getpwnam(user)
 
 groupid = grp.getgrgid(pwd_nam.pw_gid).gr_name
-groupnames = {'z5': 'lhcb', 'zp': 'atlas', 'zh': 'cms', 'vl': 'na62'}
```

Comment:
You might as well keep `atlas` and `na62` here.

Reply:
Made the changes